docker-compose.yml

```diff
@@ -18,3 +18,12 @@ services:
       - RUN_EVERY=600
     volumes:
       - .:/home/frontend/openqa-mail-notification
+  obs-package-status:
+    build:
+      dockerfile: Dockerfile
+      context: obs-package-status
+    restart: always
+    environment:
+      - RUN_EVERY=600
+    volumes:
+      - .:/home/package-bot/obs-package-status
```

Comment:
The volume directory should be: `/home/frontend/obs-package-status`, right?

Reply:
Huh, yes. Changing.